posts/templatetags/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'juancabello'
```

Comment:
This should be in a separate PR, as we did not finally decide about the format of docuentation.